commands/pack.go

```diff
@@ -0,0 +1,84 @@
+package commands
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/pack"
+	"github.com/tarantool/cartridge-cli/project"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(packCmd)
+
+	packCmd.Flags().StringVar(&projectCtx.Name, "name", "", nameFlagDoc)
+	packCmd.Flags().StringVar(&projectCtx.Version, "version", "", versionFlagDoc)
+	packCmd.Flags().StringVar(&projectCtx.Suffix, "suffix", "", suffixFlagDoc)
+	packCmd.Flags().StringVar(&projectCtx.UnitTemplatePath, "unit-template", "", unitTemplateFlagDoc)
+	packCmd.Flags().StringVar(&projectCtx.InstUnitTemplatePath, "instantiated-unit-template", "", instUnitTemplateFlagDoc)
+}
+
+var packCmd = &cobra.Command{
+	Use:   "pack TYPE [PATH]",
+	Short: "Pack application into a distributable bundle",
+	Long: `Pack application into a distributable bundle
+
+The supported types are: rpm, tgz, docker, deb`,
+	Args: cobra.RangeArgs(1, 2),
+	Run: func(cmd *cobra.Command, args []string) {
+		err := runPackCommand(cmd, args)
+		if err != nil {
+			log.Fatalf(err.Error())
+		}
+	},
+}
+
+func runPackCommand(cmd *cobra.Command, args []string) error {
+	setLogLevel()
```

Comment:
log level is general property for the whole application. Why do you still place it at start of thus/each command?

Reply:
Because it depends on parsing command arguments result.

Reply:
Maybe it's possible to add smth like `pre-run` for all commands, I will research.

Reply:
Moved into root command `PersistentPreRun`.